### PR TITLE
Feature/cue force

### PR DESCRIPTION
**Pull request by brandonabela**

### CODE REVIEW

File: Logic/bot.py [hunk 1]
```diff
@@ -92,3 +94,18 @@ def find_optimal_path(self):
...
+
+        path_distance = 0
+
+        for i, _ in enumerate(optimal_path[:-1]):
+            path_distance += self.vector.distance_from_two_points(optimal_path[i], optimal_path[i + 1])
+        
+        sorted_holes = sorted(self.holes, key=lambda tup: (-tup[1], tup[0]))
+        sorted_holes[3::] = sorted(sorted_holes[3::], key=lambda tup: (-tup[0], tup[1]))
+
+        hole_distance = self.vector.distance_from_two_points(sorted_holes[0], sorted_holes[2])
```
Comment (KeithGeorgeCiantar) [inline Logic/bot.py]: Maybe change this to max_hole_distance or max_distance?

**Review by KeithGeorgeCiantar — COMMENTED** [review]
Everything seems ok, one small change suggested.

File: Logic/bot.py (hunk 1 above)
Comment (brandonabela) [reply Logic/bot.py]: Done

**Review by KeithGeorgeCiantar — APPROVED** [review]
Everything seems ok.

**Pull request merged**